packages/sdk-artifact/src/ArtifactClient.test.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from 'earljs'
+import { ReadStream } from 'fs'
+import { readFile, rm, writeFile } from 'fs/promises'
+import { Readable } from 'stream'
+
+import { mock } from './__test/mock'
+import { ArtifactClient } from './ArtifactClient'
+import { ArtifactApi } from './networking/ArtifactApi'
+import { HttpClient } from './networking/HttpClient'
+
+describe(ArtifactClient.name, () => {
+  const fileFath = 'file.json'
+  const value = { foo: 'bar' }
+  const fileContent = JSON.stringify(value)
+  const key = 'artifact-key'
+
+  it('uploadFile', async () => {
```

Comment:
you can use `ArtifactClient.prototype.uploadFile.name` or something like this. Then refactoring will be easier :)

Reply:
Please take a look at the new solution and tell if you like it.
`ArtifactClient.prototype.`**`getArtifactUrl`**`.name` was too long and significant part was far on the right so it was hard to eyeball the tests to find the one, so I've decided to shorten it. Don't like single letter var names in general, but seems justifiable here

Reply:
This looks good :+1: